.github/workflows/build.yml

```diff
@@ -0,0 +1,52 @@
+name: "Linting and Testing"
+
+on:
+  push:
+    branches: [master,github-actions-test]
+    paths-ignore:
+    - 'README.md'
+    - '.deepsource.toml'
+
+  pull_request:
+    branches: [master]
+    paths-ignore:
+    - 'README.md'
+    - '.deepsource.toml'
+
+
+jobs:
+  build:
+
+    runs-on: ${{matrix.os}}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04]
+        python-version: [3.7,3.8,3.9]
+        test-folder : ['tests.test_multislice']
```

Comment:
`test_multisclice.py` is a file.py and a folder, thus I don't really know if this will work, but let's try!

Reply:
there's actually just `tests/tests_multislice.py`, so I will change it to `test-folder : ['tests']`. I misread the template yml I was using: https://github.com/compSPI/ioSPI/blob/master/.github/workflows/build.yml#L27

Reply:
> Actually, instead of hard-coding the import of ioSPI in the build.yml, you can install it via the simSPI.yml

I meant to work on this before, did not find the time yet. Once we are happy with a mechanism to have the various compSPI repos working with one another, we can document it in compSPI/compSPI.